### DISCUSSION TITLE
I dont see this using sass

### DISCUSSION THREAD

**Discussion by ahmu83** [tailpress/tailpress#26] [Q&A] [answered]
I dont see this using sass. Am I wrong?

**Comment by jeffreyvr** [comment, marked as answer]
It uses PostCSS, but you could use Sass if you make some changes. I just made quick demo here:

https://github.com/jeffreyvr/tailpress-sass-demo

**Reply by darylbleach** [reply]
What are the changes that I'd need to make to update the latest version to use Sass?

Thanks

**Reply by jeffreyvr** [reply]
I don't have an exact answer for you at this time, but you could use https://github.com/jeffreyvr/tailpress/tree/mix as a starting point in that case.

Then compare the `package.json` and `webpack.mix.js` files from the demo (https://github.com/jeffreyvr/tailpress-sass-demo) and change were needed.

**Reply by darylbleach** [reply]
amazing, thanks Jeffrey. I'll start going though that this week and hopefully have a working up to date version running Sass.

**Reply by jeffreyvr** [reply]
Cool @darylbleach! :)

**Reply by tdecoy** [reply]
This is useful, Jeff. Needed this too, thanks.
